src/apps/chifra/pkg/utils/utils.go

```diff
@@ -27,6 +28,13 @@ func IsTestModeServer(r *http.Request) bool {
 	return r.Header.Get("User-Agent") == "testRunner"
 }
 
+// IsServerWriter tries to cast `w` into `http.ResponseWriter`
+// and returns true if the cast was successful
+func IsServerWriter(w io.Writer) bool {
+	_, ok := w.(http.ResponseWriter)
+	return ok
+}
+
 func IsApiMode() bool {
```

Comment:
Oh. Just seeing this. I think the two (this and the above) are identical (although, I see that one needs the writer -- but I'm pretty sure we have that wherever we need it.

Upshot: don't do anything now other than note this.